File: 4-strings/1-intro/main.cpp
// Strings - C++ Example

#include <iostream>
#include <string>

int main() {
    std::string text = "Hello, World!";
    std::cout << "Original string: " << text << std::endl;

    // Concatenation
    std::string greeting = text + " Welcome!";
    std::cout << "Concatenated string: " << greeting << std::endl;

    // Substring
    std::cout << "Substring (7,5): " << text.substr(7, 5) << std::endl;

    return 0;
}

File: 4-strings/1-intro/main.java
// Strings - Java Example

class StringsIntro {

    public static void main(String[] args) {
        String text = "Hello, World!";
        System.out.println("Original string: " + text);

        // Concatenation
        String greeting = text + " Welcome!";
        System.out.println("Concatenated string: " + greeting);

        // Substring
        System.out.println("Substring (7,12): " + text.substring(7, 12));
    }
}
